code/modules/antagonists/changeling/abilities/changeling.dm

```diff
@@ -2,6 +2,9 @@
 	var/datum/abilityHolder/changeling/O = src.get_ability_holder(/datum/abilityHolder/changeling)
 	if (O)
 		return
+	var/mob/living/L = src
+	if(istype(L))
+		L.blood_id = "bloodc"
```

Comment:
Right now stable mutagen only checks for `"blood"` and not `"bloodc"`, please make it so it checks for both. Line 1582 of `Reagents-PoisonEtc.dm`.

Reply:
🤔 Good idea, I agree. But also I just realized that cowpeople are immune to getting their identity stolen via stable mutagen since they have milk blood.